packages/lesswrong/server/migrations/2022-05-06-syncEmailsFields.ts

```diff
@@ -0,0 +1,22 @@
+import { registerMigration } from './migrationUtils';
+import Users from '../../lib/collections/users/collection';
+
+registerMigration({
+  name: "syncEmailsFields",
+  dateWritten: "2022-05-06",
+  idempotent: true,
+  action: async () => {
+    const usersMissingEmail = await Users.find({email: null}).fetch()
```

Comment:
What happens if the fields populated but with a different value? I think this would cause logins to break

Reply:
We do not have users with more than one email address. For us that's hypothetical functionality that I don't think actually gets implemented in any way. I don't think `email` and `emails` ever have mismatched emails (just one or the other empty) though I need to do another check.

The plan is move towards `email` being the canonical field, though otherwise `email` and 'emails` ought to be in sync. Though I'm not sure whether I want to bother with a migration that populates `emails` for all our legacy accounts when we're moving away from that field. 

QA is going through the DB and writing queries to check the current state of all profiles.

Reply:
> Do you just not have users with more than one email address?

We only have 3 users doing that, and all of them are alts of Oliver Habryka. (I don't know why, but plan to check in with him and probably remove the alt email addresses). @Xodarap do you in fact use multiple emails on EA forum?

> What happens if the fields populated but with a different value? I think this would cause logins to break

I'm confused why this would be relevant in this case – we're selecting on users that don't have email field set to anything, so there couldn't be a mismatch.

@Xodarap you listed this as "request changes", could you be more specific on what you think should be different?